assets/scss/texts.scss

```diff
@@ -21,11 +21,16 @@ header .col {
 .text-page {
   background: $white;
   margin-top: -3rem;
-  
+  overflow-x: hidden;
+
   h1 {
     margin-bottom: 0.5rem;
     text-transform: capitalize;
   }
+  h2 {
+    margin-bottom: 0;
+    font-size: 2.125rem;
+  }
```

Comment:
This is not exactly what I meant, and I think this can result in unwanted large headings. Take a look for example at the "Chapters" header. 

The title I mean is the exhibition title on chapter and credits pages. So for example "Blues skies, red panic" on https://portaljs-test.eanadev.org/en/exhibitions/blue-skies-red-panic/fallen-giants-and-superpowers This should be 24px. 

Could you remove it here? And add the class "subtitle" to that the h2 for the exhibition title, similar to what is currently on test?

Reply:
Sorry, my mistake, I must have been looking at the wrong zeplin design which had the size as 34px. Also I had failed to add the .subtitle class to the h2 when building the AuthoredHead. Have fixed the above issues and brought it inline with the design.